tests/test_cli_claude_teams_env.py

```
@@ -183,13 +185,35 @@ def run_claude_teams(
-            print(f"FAIL: expected launcher to prepend --teammate-mode auto, got {argv_lines!r}")
+        original_arg_start = 0
+        if "--settings" in argv_lines:
+            settings_index = argv_lines.index("--settings")
+            if settings_index + 1 >= len(argv_lines):
+                print(f"FAIL: expected --settings to have a JSON value, got {argv_lines!r}")
+                raise SystemExit(1)
+            try:
+                settings = json.loads(argv_lines[settings_index + 1])
+            except json.JSONDecodeError as exc:
+                print(f"FAIL: expected --settings JSON, got {argv_lines[settings_index + 1]!r}: {exc}")
+                raise SystemExit(1)
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)